src/main/java/hudson/plugins/jacoco/JacocoBuildAction.java

```diff
@@ -359,4 +355,14 @@ public final PrintStream getLogger() {
 	    // does not run the construct and thus leaves the transient variables empty
 	    return System.out;
 	}
+
+	@Override
+	public void onAttached(Run<?, ?> run) {
+		this.owner = run;
```

Comment:
Right. Note that the `load` method will return an action whose `getBuild()` would at that time return null, but once you call `someBuild.add(theNewAction)` it gets initialized.